spring-test/common/src/main/java/io/camunda/zeebe/spring/test/proxy/ZeebeClientProxy.java

```diff
@@ -15,30 +15,30 @@
  */
 package io.camunda.zeebe.spring.test.proxy;
 
-import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.client.CamundaClient;
 import java.lang.reflect.Method;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
- * Dynamic proxy to delegate to a {@link ZeebeClient} which allows to swap the ZeebeClient object
+ * Dynamic proxy to delegate to a {@link CamundaClient} which allows to swap the ZeebeClient object
  * under the hood. This is used in test environments, where the while Zeebe engine is re-initialized
  * for every test case
  */
 public class ZeebeClientProxy extends AbstractInvocationHandler {
```

Comment:
:wrench: We could rename the class to `CamundaClientProxy` to align with the delegate's type.